_sass/layout.scss

```diff
@@ -256,11 +256,21 @@ td {
 	transition: cubic-bezier(0, 1, 0, 1);
 }
 
+
 .hideDiv {
-	display: none;
+	display:hidden;
```

Comment:
`hidden` is not a valid value for `display`. Please send out a PR to update this.